portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -31,4 +35,10 @@ definition {
 			value1 = "Displaying Form: ${formName} (Modified)");
 	}
 
+	macro viewTranslationLanguage {
+		AssertElementPresent(
+			changeLanguageTo = "${symbolName}",
```

Comment:
@wesleyroberts, why did you decide to name the variable `symbolName`?

How about using a name more related to language, like  `translationLanguage` or `language` only?

Reply:
Changes made